org.planqk.nisq.analyzer.core/src/main/java/org/planqk/nisq/analyzer/core/web/controller/AnalysisResultController.java

```diff
@@ -163,7 +167,7 @@
             Map<String, ParameterValue> typedParams =
                     ParameterValue.inferTypedParameterValue(implementation.getInputParameters(), analysisResult.getInputParameters());
 
-            ExecutionResult result = controlService.executeQuantumAlgorithmImplementation(analysisResult, typedParams);
+            ExecutionResult result = controlService.executeQuantumAlgorithmImplementation(analysisResult, typedParams, request.getBearerToken());

```

Comment:
please consider, that the bearerToken should be optional, in case a non-platform analysis result should be executed

Reply:
is it on purpose that the bearerToken is requested instead of the refresh token? Wouldn't the refreshToken be more "stressless"?